examples/data-sources/nios_dns_zone_rp/data-source.tf

```
@@ -0,0 +1,17 @@
+// Retrieve a specific Auth Zone using filters
+data "nios_dns_zone_rp" "get_auth_zone_using_filters" {
```

```suggestion
// Retrieve a specific Zone RP using filters
data "nios_dns_zone_rp" "get_zone_rp_using_filters" {
```
Please check this for all other examples in resource and data-source files